templates/default/my.cnf.main.erb

```diff
@@ -448,12 +449,14 @@ myisam_recover
 skip-innodb
 <% end %>
 
+<%- if node["percona"]["version"] < "5.7.4" %>
```

Comment:
`node["percona"]["version"]` is typically set to `5.6` or `5.7`, so this conditional will evaluate to true and keep this block in the my.cnf file